engines/xeen/dialogs/dialogs_quick_ref.cpp

```diff
@@ -40,11 +59,11 @@ void QuickReferenceDialog::execute() {
 	Common::String lines[8];
 
 	events.setCursor(0);
-
 	for (uint idx = 0; idx < (combat._globalCombat == 2 ? combat._combatParty.size() :
 			party._activeParty.size()); ++idx) {
 		Character &c = combat._globalCombat == 2 ? *combat._combatParty[idx] :
 			party._activeParty[idx];
+		const char **_tmpConditions = c._sex == FEMALE ? (const char **)Res.CONDITION_NAMES_F : (const char **)Res.CONDITION_NAMES_M;
```

Comment:
underscore prefix is usually for member variables not local ones

Reply:
Fixed